src/components/marketplace/views.tsx

```diff
@@ -171,14 +171,17 @@ const serviceDetails: IServiceDetails = {
 /* istanbul ignore next */
 function documentationTitle(service: string, url: string): string {
   const u = new URL(url);
-
-  switch (u.hostname) {
-    case 'docs.cloud.service.gov.uk':
+  const hostName =  u.hostname;
```

Comment:
I like this approach, but I think it'd be more readable as

```
swtich(hostName){
    case "docs.cloud.service.gov.uk": return "PaaS"
    case "aws.amazon.com": 
    case "docs.aws.amazon.com":
        switch(true){
            case pathName.toLowerCase().includes("elasticache"): return "Elasticache"
        }
}
```

Reply:
updated but unable to test